setup.py

```diff
@@ -577,7 +577,10 @@ def pxd(name):
         'pyxfile': '_libs/tslibs/offsets',
         'pxdfiles': ['_libs/src/util',
                      '_libs/tslibs/conversion',
-                     '_libs/tslibs/frequencies']},
+                     '_libs/tslibs/frequencies',
```

Comment:
in future prs can reduce the tslib.pyx deps I think now that lots of things are moved out.